app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const apiRouter = require('./routes/api'); 
+
+app.use(express.json());
+
+
+app.use('/api', apiRouter);
+
+
+app.use((err, req, res, next) => {
+  res.status(err.status || 500).send({ error: err.message || 'Internal Server Error' });
```

Comment:
I like the idea of having a 500 error in place as a fallback, but the custom error logic here shouldnt be added yet as we dont have a test for it to ensure it works correctly